src/ng2-filter.pipe.spec.ts

```diff
@@ -26,7 +26,13 @@ describe('Pipe: Default', () => {
     });
 
     it('Filter the nested object', () => {
-        const list = [{ a: 'e' }, { a: { b: 'e' , c: { b : 'd' }}}];
-        expect(pipe.transform(list, 'd')).toEqual([{ a: { b: 'e', c: { b : 'd' }}}]);
+        const list = [{ a: 'e' }, { a: { b: 'e', c: { b: 'd' } } }];
+        expect(pipe.transform(list, 'd')).toEqual([{ a: { b: 'e', c: { b: 'd' } } }]);
+    });
+
+    it('Filter the array of strings', () => {
+        const list = ['a', 'b', 'c'];
+        expect(pipe.transform(list, 'd')).toEqual(['']);
```

Comment:
```suggestion
        expect(pipe.transform(list, 'd')).toEqual([]);
```

This filter returns an empty array